src/test/components/__snapshots__/ThreadActivityGraph.test.js.snap

```diff
@@ -4066,292 +4001,292 @@ Array [
   Array [
     "lineTo",
     71,
-    9.954285714775324,
+    9.982857139548287,
```

Comment:
These changes are not really expressive :/ but I do have more specific tests about the activity graph at the last commit.